web/src/app/modules/views/side/modules/links-actions/services/additional-information.service.ts

```diff
@@ -17,8 +17,8 @@ import { UserToken } from '../../../../main/authentication/base/user-token';
 @Injectable()
 export class AdditionalInformationService {
 
-    private _requirement: Requirement;
-    private _model: IContainer;
+    public _requirement: Requirement;
+    public _model: IContainer;

```

Comment:
Why? I don't understand this change... wouldn't it be better to delete _requirement, _model, and _contents as they are not used?